src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Security/Permissions/WebBrowserPermission.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+namespace System.Security.Permissions 
+{
+    public enum WebBrowserPermissionLevel
+    {
+        None,
+        Safe,
+        Unrestricted
+     }
+    
+    [Serializable()]
+    sealed public class WebBrowserPermission : CodeAccessPermission, IUnrestrictedPermission 
```

Comment:
re-adding this looks like a mistake while merging?

Reply:
Sorry, my mistake